apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformerTest.java

```diff
@@ -2646,7 +2648,7 @@ FissAuditTrail.Builder getTestEntityBuilder(FissClaim.Builder claimBuilder) {
     RdaFissAuditTrail getTestEntity(RdaFissClaim claim) {
       assertEquals(1, claim.getAuditTrail().size());
       RdaFissAuditTrail answer = claim.getAuditTrail().iterator().next();
-      assertEquals("dcn", answer.getDcn());
+      assertEquals("dcn", answer.getClaimId());
```

Comment:
A little funky in that the "dcn" value is used for both, but it is just a test that the value is there.